### DISCUSSION TITLE
Pervious version to be Installed in case upgrade failed

### DISCUSSION THREAD

**Discussion by RachelXGanon** [wixtoolset/issues#6600] [Questions]
Hi,
I'm using `MajorUpgrade` element to perform upgrade, 
In case of upgrade failing, I want pervious version to be Installed.

I need only ONE version to be installed.

**How can I track the upgrade ?**
For example

- If failed to remove previous version MUST NOT install the upgrade version.
- Or If remove previous version successfully but failed in Installing the upgraded version MUST have the previous version Installed.

Regards.

**Comment by RachelXGanon** [comment]
?

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix
